src/Strategy/Erlang/ConfigParser.hs

```diff
@@ -0,0 +1,51 @@
+module Strategy.Erlang.ConfigParser (
+  parseConfig,
+  parseErlValue,
+) where
+
+import Data.Functor (void)
+import Data.Text (Text)
+import Data.Void
+import Text.Megaparsec
+import Text.Megaparsec.Char
+import qualified Text.Megaparsec.Char.Lexer as L
+
+type Parser = Parsec Void Text
+
+data ErlValue
+  = Atom Text
+  | Array [ErlValue]
+  | Pair Text ErlValue
```

Comment:
it looks like you can have arbitrary-length tuples in the dependencies input, not just length-2 text/value pairs

Reply:
Can you show or reference an example?  My understanding of erlang config syntax is that a curly-brace item is always a key-value pair.

Reply:
oh whoops, I meant to link this: https://www.rebar3.org/docs/dependencies/ . they have a bunch of git dependencies with 3- or 4-tuples. tuples are the [curly brace syntax](https://www.tutorialspoint.com/erlang/erlang_tuples.htm)